saleor/checkout/migrations/0039_alter_checkout_options.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.5 on 2021-09-09 15:32
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("checkout", "0038_merge_20210903_1048"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="checkout",
+            options={
+                "ordering": ("-last_change", "pk"),
+                "permissions": (
+                    ("manage_checkouts", "Manage checkouts"),
+                    ("handle_taxes", "Handle taxes"),
```

Comment:
Ale, we should create migration which will grant `handle_taxes` permission to all users which already have `manage_orders` or `manage_checkouts`. (See migration for `handle_payments, probably you can use it as an example).

Reply:
(This can be done i separate PR, but just wanted to let you know)